test/conftest.py

```diff
@@ -225,3 +225,15 @@ def pytest_addoption(parser: Parser) -> None:
         default=False,
         help="Disable tests marked as slow",
     )
+    parser.addoption(
+        "--cached",
+        action="store_true",
+        default=False,
+        help="Cache everything between invocations of pytest",
+    )
+    parser.addoption(
+        "--cached-automl",
```

Comment:
What does this do?

Reply:
A relic from a previous decision on what to name the option. I thought `--cached` is more straightforward.